### PR TITLE
add READMEs for VRF V1 and V2 example contracts

### PR DESCRIPTION
**Pull request by Nic0S**
Add READMEs for VRFOwnerlessConsumerExample.sol and
VRFExternalSubOwnerExample.sol that explain how to deploy and
request random words using those contracts and the Go scripts.

Also some added commands/cleanup on the V2 testnet script:
 - rename OWNER_KEY -> ACCOUNT_KEY to match the other script (which
   doesn't deal with an owner)
 - add a separate "create sub" command. The command that creates,
   funds, and authorizes a sub is incompatible with alchemy because
   it does not support notifications

### CODE REVIEW

**Review by makramkd — DISMISSED** [review]
💯 Awesome!

**Review by pramodhs2 — DISMISSED** [review]
Good work Nico, this will be super helpful.

File: core/scripts/vrfv2/testnet/main.go [hunk 1]
```diff
@@ -257,13 +254,33 @@ func main() {
...
 	case "eoa-create-sub":
+		createSubCmd := flag.NewFlagSet("eoa-create-sub", flag.ExitOnError)
+		coordinatorAddress := createSubCmd.String("coordinator-address", "", "coordinator address")
+		panicErr(createSubCmd.Parse(os.Args[2:]))
+		coordinator, err := vrf_coordinator_v2.NewVRFCoordinatorV2(common.HexToAddress(*coordinatorAddress), ec)
+		panicErr(err)
+		tx, err := coordinator.CreateSubscription(owner)
+		panicErr(err)
+		fmt.Println("Create subscription", "TX hash", tx.Hash())
+	case "add-sub-consumer":
```
[the remark this answers is not in the document]
Comment (Nic0S) [reply core/scripts/vrfv2/testnet/main.go]: Ah yeah, updated to match

File: core/scripts/vrfv2/testnet/main.go [hunk 2]
```diff
@@ -257,13 +254,33 @@ func main() {
...
+		coordinatorAddress := addSubConsCmd.String("coordinator-address", "", "coordinator address")
+		subID := addSubConsCmd.Uint64("sub-id", 0, "subID")
+		consumerAddress := addSubConsCmd.String("consumer-address", "", "consumer address")
+		panicErr(addSubConsCmd.Parse(os.Args[2:]))
+		coordinator, err := vrf_coordinator_v2.NewVRFCoordinatorV2(common.HexToAddress(*coordinatorAddress), ec)
+		panicErr(err)
+		txadd, err := coordinator.AddConsumer(owner, *subID, common.HexToAddress(*consumerAddress))
+		panicErr(err)
+		fmt.Println("Adding consumer", "TX hash", txadd.Hash())
+	case "eoa-create-fund-add-sub":
```
[the remark this answers is not in the document]
Comment (Nic0S) [reply core/scripts/vrfv2/testnet/main.go]: Done

**Review by makramkd — APPROVED** [review]

**Pull request merged**